shavee-bin/src/args.rs

```diff
@@ -170,24 +162,42 @@ impl Sargs {
                 // If PAM mode is enabled, then the user name from PAM_USER
                 // environment variable is added to the end of ZFS dataset
                 // before it is mounted.
-                // Clap has already validated PAM_USER existence.
-                dataset.push('/');
-                dataset.push_str(env::var("PAM_USER").expect(UNREACHABLE_CODE).as_str());
-                String::from("pam")
+                let user = match env::var("PAM_USER") {
+                    Ok(u) => u,
+                    Err(e) => {
+                        // FIX for "[bug] Command line parse error #7"
+                        // If PAM mode is selected but PAM_USER env doesn't exists
+                        // then return Err(error message)
+                        let error_message = e.to_string().to_owned();
+                        return Err(
+                            clap::Error::with_description(
```

Comment:
For `PAM_USER` env error message, this code converts the error message to a new `clap` message with `EmptyValue` error kind and then returns it to the calling function.